### DISCUSSION TITLE
useSendTransaction vs useContractWrite

### DISCUSSION THREAD

**Discussion by bryantoh** [wevm/wagmi#440] [Q&A]
What is the different between **useSendTransaction** and **useContractWrite**? Can I use **useSendTransaction** to call the payable contract function?

**Comment by tmm** [comment]
You can pass a payable value with `useContractWrite` by using the [`overrides`](https://wagmi.sh/docs/hooks/useContractWrite#overrides-optional=) configuration parameter.
